lib/puppet/module.rb

```diff
@@ -206,7 +206,7 @@ def read_metadata
   rescue Errno::ENOENT
     {}
   rescue JSON::JSONError => e
-    msg = "#{name} has an invalid and unparsable metadata.json file. The parse error: #{e.message}"
+    msg = _("%{name} has an invalid and unparsable metadata.json file. The parse error: %{error}") % { name: name, error: e.message }
```

Comment:
Is `metadata.json` always going to be called that regardless of what language it's in?

Reply:
@joshcooper is metadata.json file name settable based on the tasks code?

Reply:
I don't think it's settable based on https://puppet.com/docs/puppet/5.3/style_guide.html#module-metadata

Reply:
ok I'll add a comment to not translate it